format/index.md

```diff
@@ -513,6 +513,39 @@ the following members:
   data represents a resource relationship.
 * Pagination links for the primary data (as described below).
 
+Value of each member of a `links` object can be either a string
+containing the link URL or a "link object", which can contain
+the following members:
+
+* `"href"` - the string containing the link URL or an array containing multiple strings containing the link URLs,
```

Comment:
Okay, so, I think what we need to do is abort the concept of the entire links object being an array. We can say specific link relations will support an array form filled with either strings or objects any time in the future.

Since we are only reserving `self` and `related` currently, and neither of those require an array, we don't need to speak normatively about this, and the window is still open for supporting arrays on new ones later. Sound good @ethanresnick / @hhware ?